py3status/modules/insync.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+"""
+Gets the current insync status
+
+Configuration parameters:
+    format: Display format to use *(default
+        '{status}'
+        )*
+
+Format status string parameters:
+    {status} Status of Insync
+
+@author Joshua Pratt <jp10010101010000@gmail.com>
+Thanks to @author Iain Tatch <iain.tatch@gmail.com> for the script that this is based on
+@license BSD
+"""
+
+from time import time
+import subprocess
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 1
+    format = '{status}'
+
+    def check_insync(self, i3s_output_list, i3s_config):
+        status = str(subprocess.check_output(["/usr/bin/insync", "get_status"]))
```

Comment:
`/usr/bin/insync` is this always located here?  would not just `insync` work and be more portable?